### PR TITLE
contribute typescript

### PR DESCRIPTION
**Pull request by tiye**
Not tested in VS Code...

### CODE REVIEW

**Comment by tiye** [comment]
This is PR...

**Comment by lXSPandora** [comment]
I will test it

File: package.json [hunk 1]
```diff
@@ -21,10 +21,18 @@
...
+            {
+                "language": "javascriptreact",
+                "path": "./snippets/snippets.json"
+            },
+            {
+                "language": "typescript",
+                "path": "./snippets/snippets.json"
+            },
+            {
+                "language": "typescriptreact",
```
Comment (lXSPandora) [inline package.json:33]: I think this will not work

**Review by lXSPandora — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (lXSPandora) [reply package.json]: Test and on vscode and try again because the javascriptreact is an extension

Comment (jiyinyiyong) [reply package.json]: `javascriptreact` refers to `.jsx` files, `typescriptreact` refers to `.tsx` files. What did you mean by "is an extension"?

Comment (lXSPandora) [reply package.json:33]: i will try here on my vscode

Comment (linonetwo) [reply package.json:33]: Why this not work? It's from https://code.visualstudio.com/docs/languages/identifiers

Comment (linonetwo) [reply package.json:33]: Just make it support .ts and .tsx

**Pull request closed without merging**